docs/api-testing/composer.md

```diff
@@ -132,12 +651,10 @@ There are two types of input data sets you can use:
 </table>
 
 ### Unit View
-This button toggles back and forth between the Input Set and Unit Test Composer view.<br/>
-<img src={useBaseUrl('img/api-fortress/2021/01/unitView.png')} alt="Unit View"/>
-
-
-
+This button toggles back and forth between the Input Set and Unit views.
```

Comment:
I would rather say "Activates/Shows the Unit" b/c is not a "toggle" or a button that you can activate/deactivate. Same for "Input"

Reply:
@LoryApiFortress Changed to "These buttons switch between the Input Set and Unit views."